superset-frontend/src/views/CRUD/chart/ChartList.tsx

```diff
@@ -611,13 +617,27 @@ function ChartList(props: ChartListProps) {
         addDangerToast={addDangerToast}
         addSuccessToast={addSuccessToast}
         refreshData={refreshData}
+        userId={userId}
         loading={loading}
         favoriteStatus={favoriteStatus[chart.id]}
         saveFavoriteStatus={saveFavoriteStatus}
         handleBulkChartExport={handleBulkChartExport}
       />
-    );
-  }
+    ),
+    [
+      addDangerToast,
+      addSuccessToast,
+      bulkSelectEnabled,
+      favoriteStatus,
+      hasPerm,
+      loading,
+      userId,
+      refreshData,
+      saveFavoriteStatus,
+      // userKey,
```

Comment:
Can we remove this unused code?

Reply:
Yes, I tested the chart list successfully (connected and anonymously); initial commit was amended minus the above chunk: [0bb05c4](https://github.com/apache/superset/pull/20171/commits/0bb05c4a45fe3684ca2dedf68009bff89f50a499)